### ISSUE TITLE
Saisie de lettres dans les pickers

### ISSUE THREAD

**Issue by ggrimbert** [SNCFdevelopers/bootstrap-sncf#191]
Bonjour,

Il est possible de saisir n'importe quoi avec le clavier dans les champs du picker.
Mes tests se sont focalisés sur le time picker.

Pour reproduire : 
- ouvrir un composant picker; le premier champ est automatiquement sélectionné
- saisir des lettres avec le clavier

les lettres sont bien prises en compte. dans le champ; mais pas lors de la perte de focus du champ.

De la même façon, il suffit de faire tab pour re-rentrer en saisie clavier dans un champ et le modifier.

Est-il possible de bloquer cette saisie ?

**Comment by nicolaswurtz** [comment]
Hello,
Effectivement sur le time-picker on peut saisir avec le texte, mais à la validation les valeurs sont corrigées, ça ne me semble pas être trop problématique.
Si on passe l'input en type=number par ex on va avoir la déco du navigateur qui vient s'ajouter ce qui n'est pas génial non plus, il faudrait ajouter du JS pour ça, ou empêcher la saisie en ajoutant un read-only à la volée.

**Comment by ggrimbert** [comment]
Pouvoir bloquer la saisie avec un read only serait probablement la meilleure solution.
Il faudrait que ça soit un paramètre à passer, comme ça ceux qui veulent garder la saisie manuelle peuvent le faire.

**Comment by nicolaswurtz** [comment]
Oui, carrément, et ça semble assez simple à implémenter en plus (le code est assez limpide), n'hésite pas à faire une pull-request ;-)

**Comment by ggrimbert** [comment]
La pull request risque d'être un peu compliquée vu que j'ai changé le
fonctionnement de mon côté. Tu penses qu'on pourrait échanger à ce sujet ?
J'ai fait un fork que j'utilise dans mes projets et je voudrais qu'on en
parle avant de le proposer

Le ven. 14 juin 2019 à 15:31, Nicolas Wurtz <notifications@github.com> a
écrit :
